src/view/Dialog.tsx

```diff
@@ -1,26 +1,70 @@
-import React from "react";
-import './Dialog.less';
+import React, {useState} from "react";
 import {DialogData} from "../utils/typer";
-import {Innholdstittel} from "nav-frontend-typografi";
-import {Showdialog} from "./Showdialog";
+import {Innholdstittel, Undertittel} from "nav-frontend-typografi";
+import Lenke from "nav-frontend-lenker";
+import {HoyreChevron} from "nav-frontend-chevron";
+import {Checkbox, Textarea} from "nav-frontend-skjema";
+import {HenvendelseList} from "./HenvendelseList";
+import {Hovedknapp} from "nav-frontend-knapper";
+
+import './Dialog.less';
 
 interface Props {
     dialog: DialogData | null;
 }
 
 export function Dialog(props: Props) {
+    const [value, setValue] =  useState('');
+
     if (props.dialog !== null) {
         return (
-            <Showdialog dialog={props.dialog}/>
-        )
+            <div className="dialog">
+                <Lenke href="/dialog">
+                    <Undertittel>
+                        Aktivitet: {props.dialog.aktivitetId}
+                        <HoyreChevron/>
+                    </Undertittel>
+                </Lenke>
+                <div className="checkbox-block">
+                    <Checkbox
+                        label='Venter på svar fra NAV'
+                        checked={!props.dialog.venterPaSvar}
+                        className="checkbox-block__item"
+                    />
+                    <Checkbox
+                        label='Venter på svar fra bruker'
+                        checked={props.dialog.venterPaSvar}
+                        className="checkbox-block__item"
+                    />
+                </div>
+
+                {props.dialog.henvendelser === undefined? null :<HenvendelseList henvendelseDataList={props.dialog.henvendelser}/>}
```

Comment:
Jeg fikk en kommentar på denne linjen fra @nutgaard om at sjekking for null bør være inne i komponenten som kalles.

Reply:
@Y130311-NAV Hvorfor kan `DialogData` være null? Fra app er det et element fra array som sendes og det bør ikke være null der. Vil tro at det er typen på props som bør endres til å ikke kunne inkludere null

Reply:
@Y130311-NAV Ops, jeg var visst litt for rask :) Ingeborg har ikke endret på logikken, kun kopiert koden til en ny komponent. Kan du lage en PR på det du nevner her?

Reply:
Gjort: Pull Request med "FO-2348: Flytte Undefined-sjekk inn i HenvendelseList, Rette layout detaljer (borders kun nederst, tastefeil : app_body og henvendelse-list)" venter på review/godkjenning.